File: proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/IdentityTooLong.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class IdentityTooLong : Exception
    {
        public IdentityTooLong(int actLen, int expLen)
            : base(String.Format("The length should short than {0}, but it is {1}",
                expLen, actLen))
        {
        }
    }
}

File: branches/V1.0/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/IdentityTooLong.java
package com.jasonzqshen.familyaccounting.core.exception;

public class IdentityTooLong extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1241326352948137528L;

	public IdentityTooLong(int actLen, int expLen) {
		super(String.format("The length should short than %d, but it is %d",
				expLen, actLen));
	}
}
